docs/UserGuide/System Tools/NodeTool.md

```diff
@@ -0,0 +1,288 @@
+| --- | --- | --- |
+|-m | --metadata	Query metadata partition, the default is query data partition|	-m |
+|-path | --path 	Required parameter, the path to be queried. If the path has no corresponding storage group, the query fails|	-path root.guangzhou.d1|
+|-st | --StartTime	The system uses the current partition time by default|	-st 1576724778159 |
+|-et | --EndTime	It is used when querying data partition.<br>The end time is the current system time by default. <br> If the end time is less than the start time, the end time is the start time by default|-et 1576724778159 |
+
+2.Output
+
+> The output is a multi line string, and each line of string is a key value pair, where the key represents the partition,
+> and the value represents the data group in the format of \< key -> value>.
+
+3.Examples
+
+> Suppose that the current cluster runs on three nodes: 127.0.0.1:9003:40010, 127.0.0.1:9005:40012 and 127.0.0.1:9004:40011.
+> 
+> The number of copies is 2 and there are 3 storage groups:{ root.beijing , root.shanghai , root.guangzhou}.
+
++ Partition of query data (default time interval, time dimension is partitioned by day)
+
+Linux and MacOS：
+```
+Shell > ./bin/nodetool.sh -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1
+```
+Windows：
+```
+Shell > \bin\nodetool.bat -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1
+```
+
+Press enter to execute the command. 
+
+The output of the example instruction is as follows:
+```
+DATA<root.guangzhou.d1, 1576723735188, 1576723735188>	->	[127.0.0.1:9003:40010, 127.0.0.1:9004:40011]
+```
+
++ Partition of query data (specified time interval, time dimension is partitioned by day)
+
+
+Linux and MacOS：
+```
+Shell > ./bin/nodetool.sh -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1 -st 1576624778159 -et 1576724778159
+```
+Windows：
+```
+Shell > \bin\nodetool.bat -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1 -st 1576624778159 -et 1576724778159
+```
+
+Press enter to execute the command. 
+
+The output of the example instruction is as follows:
+```
+DATA<root.guangzhou.d1, 1576627200000, 1576713599999>	->	[127.0.0.1:9005:40012, 127.0.0.1:9003:40010] 
+DATA<root.guangzhou.d1, 1576713600000, 1576724778159>	->	[127.0.0.1:9003:40010, 127.0.0.1:9004:40011] 
+DATA<root.guangzhou.d1, 1576624778159, 1576627199999>	->	[127.0.0.1:9004:40011, 127.0.0.1:9005:40012]
+```
+
++ Query metadata partition
+
+Linux and MacOS：
+```
+Shell > ./bin/nodetool.sh -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1 -m
+```
+Windows：
+```
+Shell > \bin\nodetool.bat -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1 -m
+```
+
+Press enter to execute the command. 
+
+The output of the example instruction is as follows:
+```
+DATA<root.guangzhou.d1, 1576723735188, 1576723735188>	->	[127.0.0.1:9003:40010, 127.0.0.1:9004:40011]
+```
+The above output shows that the data partition to which root.t1.d1 belongs contains two nodes,
+of which 127.0.0.1:9003:40010 is the leader node.
```

Comment:
is this first node a leader node or header node?

Reply:
header, thank you point at that.